pkg/kclient/deployments.go

```diff
@@ -115,6 +116,40 @@ func (c *Client) ListDeployments(selector string) (*appsv1.DeploymentList, error
 	})
 }
 
+func (c *Client) WaitForPodNotRunning(name string) error {
```

Comment:
Please add some comments for the function

Reply:
done